src/main/java/org/openRealmOfStars/starMap/Route.java

```diff
@@ -180,10 +113,9 @@ public boolean isFixing() {
    */
   public void makeNextMove() {
     for (int i = 0; i < ftlSpeed; i++) {
-      if (distance > 0) {
-        startX = startX + mx;
-        startY = startY + my;
-        distance--;
+      if (getDistance() > 0) {
+        startX = startX + getMx();
```

Comment:
Yes, they are different. I created a new branch which show it:
https://github.com/tuomount/Open-Realms-of-Stars/commit/a873ae46621c79e41e0e3373fbcd07b29e84152a